lib/inquirer-prompt.js

```diff
@@ -3,21 +3,20 @@ const chalk = require('chalk');
 
 module.exports = function(questions, config) {
 	inquirer.prompt(questions).ui.process
-  .reduce(function (newOpts, ans) {
-	return attachAnswers(newOpts, ans);
-}, Object.assign(config, require('./utils/initial-config')))
-  .flatMap(newOpts => {
-	const parser = require('./parser/index');
-	return parser(null, checkEmptyAnswers(newOpts));
-})
-  .subscribeOnError( (err) => process.stdout.write(err.toString()));
+	.reduce(function (newOpts, ans) {
+		return attachAnswers(newOpts, ans);
+	}, Object.assign(config, require('./utils/initial-config')))
+	.flatMap(newOpts => {
+		const parser = require('./parser/index');
```

Comment:
Scope and that we're not mutating the new config returned. Also, nice convention from rx point of view: https://github.com/Reactive-Extensions/RxJS/issues/1414